### PR TITLE
Grad developer

### PR DESCRIPTION
**Pull request by vatsalyagoel**
Adding graduate developer mad skillz

### CODE REVIEW

**Comment by rbanks54** [comment]
Will need some of the Melbourne crew to chip in on this one, based on the experiences there.

File: Consulting.md [hunk 1]
```diff
@@ -1,4 +1,24 @@
 # Consulting Roles
+## Graduate Developer
+> I am a committed learner who will prove to be a valuable asset absorbing Readifarian values.
```
Comment (jhsowter) [inline Consulting.md]: I am questioning the grammar here. Is "asset absorbing values" correct? I think I would be more comfortable with a comma so it reads "... asset, absorbing Readifarian values." 

I also think "absorbing" suggests a passive way of learning. 

How about simply "I am a committed student of Readifarian values"?

File: Consulting.md [hunk 2]
```diff
@@ -1,4 +1,24 @@
...
+- I am learning how to use a build and delivery pipeline.
+- I am learning how to convert client requirements into deliverable units of work.
+- I am learning to be self-sufficient when using my toolchain.
+- I understand the value add of different types of testing.
+- I am learning to implement design patters that are widely used in the industry.
+- I am learning about the infrastructure my code runs on.
+- I like to seek clarification on new technologies that I am learning.
+
+#### I show great enthusiasm and initiatives.
+- I understand the importance of PD and recognise the value it plays in my progress.
```
Comment (jhsowter) [inline Consulting.md]: Make this "recognise the role is plays" or "recognise the value it adds". And now I think about it, understand and recognise are practically synonyms here so let's just make it "I understand the importance of PD and it's role in my progress."

File: Consulting.md [hunk 3]
```diff
@@ -1,4 +1,24 @@
...
+- I am learning the concepts of agile development and will thrive to apply within the project I am allocated to.
```
Comment (jhsowter) [inline Consulting.md]: Replace "thrive" with "try".

File: Consulting.md (hunk 1 above)
Comment (kjacobsen) [inline Consulting.md]: Agreed, the word "absorbing" doesn't sound right here. I think we should go with Jack's wording.

File: Consulting.md [hunk 4]
```diff
@@ -1,4 +1,24 @@
 # Consulting Roles
+## Graduate Developer
+> I am a committed learner who will prove to be a valuable asset absorbing Readifarian values.
+
+#### I write good quality code that can be shipped to production.
+- I am learning about automated testing.
+- I am learning how to use a build and delivery pipeline.
+- I am learning how to convert client requirements into deliverable units of work.
+- I am learning to be self-sufficient when using my toolchain.
+- I understand the value add of different types of testing.
```
Comment (kjacobsen) [inline Consulting.md]: I suspect that the idea was to also cover the different types (unit, integration, UAT) , along with automated processes.

Perhaps we can combine the two.

File: Consulting.md [hunk 5]
```diff
@@ -1,4 +1,24 @@
...
+- I am learning to implement design patters that are widely used in the industry.
+- I am learning about the infrastructure my code runs on.
+- I like to seek clarification on new technologies that I am learning.
```
Comment (kjacobsen) [inline Consulting.md]: I think we need to tweak this one, like Jack, I am not sure exactly what the point is.

Are you saying that if there is a technology (new or old) that you don't know, you will ask for help?

File: Consulting.md [hunk 6]
```diff
@@ -1,4 +1,24 @@
...
+
+#### I show great enthusiasm and initiatives.
+- I understand the importance of PD and recognise the value it plays in my progress.
+- I am learning the concepts of agile development and will thrive to apply within the project I am allocated to.
+- I am self-directed in my learning of technologies that interest me.
```
Comment (kjacobsen) [inline Consulting.md]: This line is good, but I would suggest you may need some guidance on determining what PD you should be aiming for.

**Comment by vatsalyagoel** [comment]
Implemented changes. Please review @jhsowter  @kjacobsen

**Comment by jhsowter** [comment]
This is good to go I think.

**Pull request merged**